File: seo-agent/check_tasks.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv('.env')

url: str = os.environ.get("SUPABASE_URL")
key: str = os.environ.get("SUPABASE_SERVICE_ROLE_KEY")
supabase: Client = create_client(url, key)

res = supabase.table("tasks").select("id, title, session_id").order("created_at", desc=True).limit(5).execute()
print(res.data)

File: agentic-seo/check_tasks.ts
import { createClient } from '@supabase/supabase-js'
import dotenv from 'dotenv'

dotenv.config({ path: '.env.local' })

const supabase = createClient(
  process.env.NEXT_PUBLIC_SUPABASE_URL!,
  process.env.SUPABASE_SERVICE_ROLE_KEY!
)

async function check() {
  const { data } = await supabase.from('tasks').select('id, title, session_id').order('created_at', { ascending: false }).limit(5)
  console.log(data)
}
check()
